spec/rails_helper.rb

```diff
@@ -85,20 +120,31 @@
     raise StandardError.new "\"#{example.full_description}\" in #{example.location} timed out."
   end
 
-  config.around :each, :disable_bullet do |example|
-    Bullet.raise = false
-    example.run
-    Bullet.raise = true
-  end
-
-  def pre_transition_aged_youth_age
-    Date.current - CasaCase::TRANSITION_AGE.years
-  end
+  # NOTE: not applicable currently, left to show how to skip bullet errrors
+  # config.around :each, :disable_bullet do |example|
+  #   Bullet.raise = false
+  #   example.run
+  #   Bullet.raise = true
+  # end
 
   config.around do |example|
     Capybara.server_port = 7654 + ENV["TEST_ENV_NUMBER"].to_i
     example.run
   end
 
-  config.filter_run_excluding :ci_only unless ENV["GITHUB_ACTIONS"]
+  config.filter_run_excluding :ci_only unless ci_environment
```

Comment:
We don't use this tag currently, we could just remove it? Only use case I can think of is coverage reports, and we have an env flag for that (spec helper).